vdr/api/v2/api.go

```diff
@@ -166,12 +168,58 @@ func (w Wrapper) UpdateService(ctx context.Context, request UpdateServiceRequest
 	return UpdateService200JSONResponse(*newService), nil
 }
 
-func (w Wrapper) AddVerificationMethod(ctx context.Context, request AddVerificationMethodRequestObject) (AddVerificationMethodResponseObject, error) {
+func (w *Wrapper) ResolveServiceEndpointByType(_ context.Context, request ResolveServiceEndpointByTypeRequestObject) (ResolveServiceEndpointByTypeResponseObject, error) {
```

Comment:
did spec allows multiple services with the same type, is this handled correctly everywhere (else)?

Reply:
We now have a search/filter-style operation that can return multiple services.